src/javascript/app/pages/cashier/cashier.js

```diff
@@ -65,14 +69,60 @@ const Cashier = (() => {
         });
     };
 
+    const showCurrentCurrency = (currency, has_no_mt5, has_no_tx) => {
+        const el_acc_currency     = getElementById('account_currency');
+        const el_currency_image   = getElementById('account_currency_img');
+        const el_current_currency = getElementById('account_currency_current');
+        const el_current_hint     = getElementById('account_currency_hint');
+
+        const missingCriteria     = (has_mt5, has_tx) => {
```

Comment:
extra space before `=`, we don't need to align it with above since there's an empty line between them